azurerm/internal/services/recoveryservices/tests/resource_arm_backup_policy_vm_test.go

```diff
@@ -535,3 +558,25 @@ resource "azurerm_backup_policy_vm" "test" {
 }
 `, template, data.RandomInteger)
 }
+
+func testAccAzureRMBackupProtectionPolicyVM_basicDailyWithInstantRestoreRetentionRange(data acceptance.TestData) string {
+	template := testAccAzureRMBackupProtectionPolicyVM_template(data)
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_backup_policy_vm" "test" {
+  name                            = "acctest-%d"
```

Comment:
```suggestion
  name                            = "acctest-BPVM-%d"
```

we like to make sure capitals are handles correctly

Reply:
It looks like per [this documentation](https://docs.microsoft.com/en-us/azure/backup/backup-instant-restore-capability) the range is between 1 and 5 days, I'll update the validation function accordingly.